### DISCUSSION TITLE
How to estimate gas using Wagmi's contactwrite and rewrite contractwrite to send transactions based on the gas estimate multiplied by 1.5.

### DISCUSSION THREAD

**Discussion by GalaxySciTech** [wevm/wagmi#1943] [Q&A]
How to estimate gas using Wagmi's contactwrite and rewrite contractwrite to send transactions based on the gas estimate multiplied by 1.5.

**Comment by phatth2016** [comment]
you can try:
      const estimateGasData = await Contract?.estimateGas?.['ContractFunc'](params1, params2, ....)
